src/pages/en/migrate.mdx

```diff
@@ -95,7 +95,7 @@ When upgrading, please visually inspect your site output to make sure everything
 
 Astro no longer supports components or JSX expressions in Markdown pages by default. For long-term support you should migrate to the [`@astrojs/mdx`](/en/guides/integrations-guide/mdx/) integration.
 
-To make migrating easier, a new [legacy flag](/en/reference/configuration-reference/#legacyastroflavoredmarkdown) can be used to re-enable previous Markdown features.
+To make migrating easier, a new `legacy.astroFlavoredMarkdown` flag can be used to re-enable previous Markdown features.
```

Comment:
```suggestion
To make migrating easier, a new `legacy.astroFlavoredMarkdown` flag can be used to re-enable previous Markdown features. (Removed in v2.0)
```
I was thinking of some kind of note here. Either as above, or 

> a new `legacy.astroFlavoredMarkdown` flag (removed in v2.0)

Thoughs?

Reply:
Even though we’re going to delete this all? I’m not opposed though — doesn’t do any harm.

Reply:
Yes, still my preference! (FWIW)

Reply:
Edited your suggestion to use your second suggestion 👍